### PR TITLE
[SwitchOrganization] Snackbar optimization

### PR DESCRIPTION
**Pull request by sumitra19jha**
Issue: #175 
The aim of this pull is to make sure the error message is displayed in the snackbar without
1. Breaking the widget
2. In a user appealing way

**Current Behaviour:**
Screenshot 2021-03-15 at 3 46 32 PM

**Expected Behaviour:**
Save button to be removed or snackbar to be optimized

**Steps to create Issue:**

1. Login/Signup
2. Remove Yourself from all organization
3. Now restart the app and navigate to the Profile Screen
4. In Profile screen navigate to Switch Organization
5. Press Save

### CODE REVIEW

**Comment by sumitra19jha** [comment]
@Sagar2366 Can you please review this.
The error still exists

**Comment by DeltaHarbinger** [comment]
Hey @sumitra19jha, can you provide a video with the current visuals of your fix, or provide steps to replicate the issue? I'll need this to properly assess your fix

**Comment by sumitra19jha** [comment]
> Hey @sumitra19jha, can you provide a video with the current visuals of your fix, or provide steps to replicate the issue? I'll need this to properly assess your fix

Sure.

https://user-images.githubusercontent.com/35868598/111071811-dcbe8d80-84fd-11eb-8620-48da16ea2412.mp4

**Comment by sumitra19jha** [comment]
@DeltaHarbinger Do let me know about your feedback and if you are thinking of any further optimisation.

**Comment by Sagar2366** [comment]
@sumitra19jha, remove switch organization option from list if user is not part of any organization to avoid further issues.

**Comment by Sagar2366** [comment]
Also, please check comments in related issue https://github.com/PalisadoesFoundation/talawa/issues/175 and add necessary details.

**Comment by sumitra19jha** [comment]
ok @Sagar2366. Will take care of that and working on removing switch organization if user is not part of any organization

**Comment by Sagar2366** [comment]
> ok @Sagar2366. Will take care of that and working on removing switch organization if user is not part of any organization

Update PR and issue description for the same.

**Comment by sumitra19jha** [comment]
> Also, please check comments in related issue #175 and add necessary details.

I made the changes in PR #315 
I haven't remove the `Switch Organization` button as we have to query data first of all organization and then we can decide there. This make UI building process expensive. As Switch organization page already has organization list so I removed the save button there when no organization is present.

**Comment by sumitra19jha** [comment]
@Sagar2366 PR #329 solve it completely.

**Comment by Sagar2366** [comment]
@sumitra19jha  why are you creating multiple PR's for same issue, it is really difficult to track.
How https://github.com/PalisadoesFoundation/talawa/pull/329 is different from this ? Can you please merge those changes as part of this PR and share it for review ?

**Comment by sumitra19jha** [comment]
@Sagar2366 Please check the updates here

File: lib/views/pages/organization/profile_page.dart [hunk 1]
```diff
@@ -226,21 +228,23 @@ class _ProfilePageState extends State<ProfilePage> {
...
-                                Icons.compare_arrows,
-                                color: UIData.secondaryColor,
-                              ),
-                              onTap: () {
-                                pushNewScreen(
-                                  context,
-                                  screen: SwitchOrganization(),
-                                );
-                              }),
+                          org.length == 0
```
Comment (Sagar2366) [inline lib/views/pages/organization/profile_page.dart:231]: Good work @sumitra19jha.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**